src/_samples/index.md

```diff
@@ -252,26 +268,37 @@ abstract class Describable {
 }
 {% endprettify %}
 
-Any class extending `Describable` has the `describeWithEmphasis()` method, which calls the extender's implementation of `describe()`.
+Any class extending `Describable` or using it as a mixin has the `describeWithEmphasis()` method, which calls the extender's / base class implementation of `describe()`.
+
+<?code-excerpt "lib/samples/spacecraft.dart (abstract use)" replace="/abstract/[!$&!]/g"?>
+{% prettify dart %}
+class Probe extends Spacecraft with Describable {
+  Probe(String name) : super(name, new DateTime.now());
+}
+{% endprettify %}
 
 [Read more](/guides/language/language-tour#abstract-classes) about abstract classes and methods.
 
 ## Async
 
 You can avoid callback hell and make your code much more readable by using `async` and `await`.
 
+<?code-excerpt "test/samples_test.dart (async)" replace="/async/[!$&!]/g"?>
 {% prettify dart %}
-Future<Null> printWithDelay(String message) async {
-  await new Future.delayed(const Duration(seconds: 1));
+const aSecond = const Duration(seconds: 1);
+// ···
+Future<Null> printWithDelay(String message) [!async!] {
+  await new Future.delayed(aSecond);
   print(message);
 }
 {% endprettify %}
 
 The code above is equivalent to:
```

Comment:
code -> method

(because you added the code for const aSecond)

Reply:
Good catch. Fixed.